pkg/controller/scyllacluster/resource.go

```diff
@@ -689,7 +689,7 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, existing
 								PeriodSeconds:    int32(10),
 								ProbeHandler: corev1.ProbeHandler{
 									HTTPGet: &corev1.HTTPGetAction{
-										Port: intstr.FromInt(naming.ProbePort),
+										Port: intstr.FromInt(naming.ScyllaDBAPIStatusProbePort),
```

Comment:
These that are handled by other container could use a comment explaining why.

Reply:
done